nodeadm/internal/kubelet/config.go

```diff
@@ -230,7 +230,7 @@ func (ksc *kubeletConfig) withCloudProvider(cfg *api.NodeConfig, flags map[strin
 
 	// use ec2 instance-id as node hostname which is unique, stable, and incurs
 	// no additional requests
-	flags["hostname-override"] = cfg.Status.Instance.ID
+	// flags["hostname-override"] = cfg.Status.Instance.ID
```

Comment:
We'll need to override the hostname to the privateDnsName, or we'll run into the mess we did on AL2

Reply:
will the duration of the preview last for the related changes on instance-id names, or do we need to properly support this for X number of releases?